build.gradle

```diff
@@ -7,15 +7,21 @@ buildscript {
         springBootVersion = '2.4.5'
         blackDuckCommonVersion = '59.0.4'
         junitPlatformCustomTestTags = 'docker, battery'
+        javaTargetCompatibility = 8
+        javaSourceCompatibility = 8
     }
 
     apply from: 'https://raw.githubusercontent.com/blackducksoftware/integration-resources/master/gradle_common/buildscript-repositories.gradle', to: buildscript
-    apply from: 'https://raw.githubusercontent.com/blackducksoftware/integration-resources/master/gradle_common/buildscript-dependencies.gradle', to: buildscript
+
     dependencies {
         classpath "org.springframework.boot:spring-boot-gradle-plugin:${springBootVersion}"
     }
 }
 
+plugins {
+    id 'org.barfuin.gradle.taskinfo' version '1.3.0'
+}
+
```

Comment:
@DMaxfield-BDS is this needed? It was apart of your example repos: https://github.com/blackducksoftware/cgp-sample-projects/blob/main/buildSrc-gradle-7/build.gradle#L9

Reply:
No, it is not needed.